stubs/docutils/docutils/nodes.pyi

```diff
@@ -129,19 +139,76 @@ class Element(Node):
     def __add__(self, other: list[Node]) -> list[Node]: ...
     def __radd__(self, other: list[Node]) -> list[Node]: ...
     def __iadd__(self, other: Node | Iterable[Node]) -> Self: ...
-    def copy(self) -> Self: ...
-    def deepcopy(self) -> Self: ...
-    def pformat(self, indent: str = "    ", level: int = 0) -> str: ...
     def astext(self) -> str: ...
-    def index(self, item: Node, start: int = 0, stop: int = sys.maxsize) -> int: ...
-    def remove(self, item: Node) -> None: ...
+    def non_default_attributes(self) -> dict[str, Any]: ...
+    def attlist(self) -> list[tuple[str, Any]]: ...
+    @overload
+    def get(self, key: str) -> Any: ...
+    @overload
+    def get(self, key: str, failobj: _T) -> _T: ...
+    def hasattr(self, attr: str) -> bool: ...
+    def delattr(self, attr: str) -> None: ...
+    @overload
+    def setdefault(self, key: str) -> Any: ...
+    @overload
+    def setdefault(self, key: str, failobj: _T) -> Any | _T: ...
+    def has_key(self, attr: str) -> bool: ...
+    def get_language_code(self, fallback: str = "") -> str: ...
+    def append(self, item: Node) -> None: ...
+    def extend(self, item: Iterable[Node]) -> None: ...
     def insert(self, index: SupportsIndex, item: Node | Iterable[Node] | None) -> None: ...
+    def pop(self, i: int = -1) -> Node: ...
+    def remove(self, item: Node) -> None: ...
+    def index(self, item: Node, start: int = 0, stop: int = sys.maxsize) -> int: ...
     def previous_sibling(self) -> Node | None: ...
-    def __getattr__(self, name: str, /) -> Incomplete: ...
+    def is_not_default(self, key: str) -> Literal[0, 1]: ...
```

Comment:
I wonder whether it's better to just return `bool` here, as this is clearly how it's intended to be used. Using `Literal[0, 1]` will prevent the return value from being used as a `bool`, which works at runtime.